content/zh/docs/project-user-guide/application-workloads/deployments.md

```diff
@@ -1,143 +1,144 @@
 
     {{< notice note >}}
 
-- Remember to press **Enter** on your keyboard after you input an image name in the search bar.
-- If you want to use your private image repository, you should [create an Image Registry Secret](../../configuration/image-registry/) first in **Secrets** under **Configurations**.
+- 在搜索栏输入镜像名称后，请记得按键盘上的**回车键**。
+- 如果想使用您的私有镜像仓库，您应该先通过**配置中心**下面的**密钥**[创建镜像仓库密钥](../../../project-user-guide/configuration/image-registry/)。
 
     {{</ notice >}}
 
-4. Set requests and limits for CPU and memory resources based on your needs. For more information, see [Resource Request and Resource Limit in Container Image Settings](../container-image-settings/#add-container-image).
+4. 根据您的需求设置 CPU 和内存的资源请求和限制。有关更多信息，请参见[容器镜像设置中关于资源请求和资源限制的内容](../../../project-user-guide/application-workloads/container-image-settings/#添加容器镜像)。
 
-    ![deployments](/images/docs/project-user-guide/workloads/deployments_form_2_container_2.jpg)
+    ![资源设置](/images/docs/zh-cn/project-user-guide/application-workloads/deployments/resource-setting.PNG)
 
-5. Click **Use Default Ports** for **Port Settings** or you can customize **Protocol**, **Name** and **Container Port**.
+5. **端口设置**请点击**使用默认端口**，或者您可以自定义**协议**、**名称**和**容器端口**。
 
-6. Select a policy for image pulling from the drop-down menu. For more information, see [Image Pull Policy in Container Image Settings](../container-image-settings/#add-container-image).
+6. 在下拉菜单中选择镜像拉取策略。有关更多信息，请参见[容器镜像设置中关于镜像拉取策略的内容](../../../project-user-guide/application-workloads/container-image-settings/#添加容器镜像)。
 
-7. For other settings (**Health Checker**, **Start Command**, **Environment Variables**, **Container Security Context** and **Sync Host Timezone**), you can configure them on the dashboard as well. For more information, see detailed explanations of these properties in [Container Image Settings](../container-image-settings/#add-container-image). When you finish, click **√** in the bottom right corner to continue.
+7. 对于其他设置（**健康检查器**、**启动命令**、**环境变量**、**容器 Security Context** 以及**同步主机时区**），您也可以在仪表板上配置它们。有关更多信息，请参见[容器镜像设置](../../../project-user-guide/application-workloads/container-image-settings/#添加容器镜像)中对这些属性的详细说明。操作完成后，点击右下角的 **√** 继续。
 
-8. Select an update strategy from the drop-down menu. It is recommended you choose **RollingUpdate**. For more information, see [Update Strategy](../container-image-settings/#update-strategy).
+8. 在下拉菜单中选择更新策略。建议您选择**滚动更新**。有关更多信息，请参见[更新策略](../../../project-user-guide/application-workloads/container-image-settings/#更新策略)。
 
-9. Select a deployment mode. For more information, see [Deployment Mode](../container-image-settings/#deployment-mode).
+9. 选择部署模式。有关更多信息，请参见[部署模式](../../../project-user-guide/application-workloads/container-image-settings/#部署模式)。
 
-10. Click **Next** to go to the next step when you finish setting the container image.
+10. 完成容器镜像设置后，点击**下一步**进行下一步操作。
 
-### Step 4: Mount Volumes
+### 步骤 4：挂载存储
```

Comment:
挂载存储
->
挂载存储卷

Reply:
I translated **Mount Volumes** into **挂载存储** to keep it consistent with the UI on ZH tab. So is it the case that "volume" should always be translated into "存储卷"?

Reply:
Technically, "卷" is the most accurate expression. Please use "存储卷" here. This part is about mounting volumes to the workload. The Chinese UI is not properly rendered while it may have sth to do with the space in UI.

Reply:
Got it.